Breeze.Sharp/Json/JsonQueryExpressionVisitor.cs

```diff
@@ -274,15 +274,15 @@ public class JsonQueryExpressionVisitor : ExpressionVisitor {
         order = " " + order.Trim();
       }
 
-      //lambdaExpression = (LambdaExpression)Evaluator.PartialEval(lambdaExpression);
+      MemberExpression body = lambdaExpression.Body is MemberExpression ?
+        (MemberExpression)lambdaExpression.Body : ((UnaryExpression)lambdaExpression.Body).Operand as MemberExpression;
 
-      MemberExpression body = lambdaExpression.Body as MemberExpression;
       if (body != null) {
-        if (string.IsNullOrEmpty(OrderBy)) {
-          OrderBy = string.Format("{0}{1}", body.Member.Name, order);
-        } else {
-          OrderBy = string.Format("{0}, {1}{2}", OrderBy, body.Member.Name, order);
```

Comment:
Wrong implementation of OrderBy yesterday I discovered that Parameters implementation is even missing